src/rotator_library/error_handler.py

```
@@ -478,101 +478,99 @@ def _extract_retry_from_json_body(json_text: str) -> Optional[int]:
-        retry_header = headers.get("retry-after") or headers.get("Retry-After")
-        if retry_header:
-            try:
-                return int(retry_header)  # Assumes seconds format
-            except ValueError:
-                pass  # Might be HTTP date format, skip for now
+                        reset_ts = int(candidate)
+                        delta = reset_ts - int(time.time())
+                        if delta > 0:
+                            return delta
+                    except Exception:
+                        pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)